doc/whatsnew/2.2.rst

```diff
@@ -19,4 +19,7 @@ Other Changes
 * `unnecessary-pass` is now also emitted when a function or class contains only docstring and pass statement, 
   in which case, docstring is enough for empty definition.
 
+* fix false positive ``undefined-variable`` and ``used-before-assignment`` with nonlocal keyword usage.
```

Comment:
Ditto. This could be:
```
Fixed a false positive for ``...`` and ``...` which happens when a `nonlocal` keyword is used and the variable is defined outside the scope where the nonlocal was used

Closes #...
```